dts/bindings/base/pm.yaml

```diff
@@ -16,15 +16,6 @@ properties:
       Wake up capable devices are disabled (interruptions will not wake up
       the system) by default but they can be enabled at runtime if necessary.
 
-  power-domain:
-    type: phandle
-    description: |
-
-      Power domain the device belongs to.
-
-      The device will be notified when the power domain it belongs to is either
```

Comment:
Should this comment be migrated across?

Reply:
This is an implementation detail, not related to DT. DT just describes the domain connections, not how they're handled.